samples/SkiaJsSample/settings.gradle.kts

```
@@ -10,3 +10,10 @@ pluginManagement {
 }
 rootProject.name = "SkiaJsSample"
 
+if (System.getenv("SKIKO_COMPOSITE_BUILD") == "1" || System.getProperty("skikoCompositeBuild") == "true") {
```

```suggestion
// This section is needed to use Skiko from sources instead as a maven artifact. If you use Skiko in your own project outside of this repository, you don't this section.
if (File("../../skiko").exists()) {
```
I assume, when we develop Skiko - there is no point to use Skiko from maven repo. And if users just copy the example, it will work too.

This change will simplify the whole process:
- we can just run the example with a simple Gradle command without additional properties (`./gradlew run`). It will work as in our use case, and in user's use case (when they copy the example)
- we can open the example in IDEA without adding a global SKIKO_COMPOSITE_BUILD (I just checked the AWT sample, highlighting and debugging work)

If we do this change, we can (and should) simplify the Readme and the CI file - there will be no need to call `publishAllSkikoJvmRuntimeToMavenLocal` and define `skiko.version`.

Also, because of this, we can remove `skiko.version` property completely, and just use `0.7.41` constant instead of it.

(we can discuss this online, to sync on the final decision)

---

It's a good suggestion. But we have problems with this case.

`includeBuild("skiko")` provides only `org.jetbrains.skiko:skiko:1.2.3` artifact.

When we run task `publishAllSkikoJvmRuntimeToMavenLocal`, under the hood this task downloads and packs skia inside jar and saves to mavenLocal several additional dependencies `org.jetbrains.skiko:skiko-awt-runtime-macos-arm64:1.2.3`, `org.jetbrains.skiko:skiko-awt-runtime-macos-x64:0.0.0`, `org.jetbrains.skiko:skiko-android-runtime-arm64:1.2.3` and others.

Also, we have task `publishSkikoWasmRuntimePublicationToMavenLocal` to save `org.jetbrains.skiko:skiko-js-wasm-runtime:1.2.3`

I think we also can solve this problem, but I don't know how...
For this case, we need to rewrite publishing tasks and also provide modules instead of jars: https://github.com/JetBrains/skiko/blob/90fb5988e064d583daa920791418c3b7a69a154b/skiko/build.gradle.kts#L1232